src/together/resources/finetune.py

```diff
@@ -140,7 +165,11 @@ def create(
             batch_size (int, optional): Batch size for fine-tuning. Defaults to max.
             learning_rate (float, optional): Learning rate multiplier to use for training
                 Defaults to 0.00001.
+            min_lr_ratio (float, optional): Ending learning rate as a percentage of the initial learning rate for
+                learning rate scheduler. Defaults to 0.0.
             warmup_ratio (float, optional): Warmup ratio for learning rate scheduler.
+            max_grad_norm (float, optional): Max gradient norm. Defaults to None.
+            weight_decay (float, optional): Weight decay. Defaults to None.
```

Comment:
Should we allow it to be None? Probably safer to only permit floating point values and use 0 by default

Reply:
Updated to only float